loader/app/service/tree_parser.py

```diff
@@ -0,0 +1,72 @@
+"""Tree result parser for backend API lookup results.
+
+See backend's app.api.api_v1.routers.lookups.tree_table_to_json
+
+A tree had 0/1/many nodes.
+A node typically looks like this:
+
+    {
```

Comment:
I don't think we can assume a single root node, so I made the return type for "tree-ish" lookups a list of objects. the "nodes" at the top level are the top level elements (e.g. world bank regions)

Reply:
Yeah, it was a silly example. The parser iterates the whole tree, though.